src/server/db_slice.cc

```diff
@@ -605,6 +613,7 @@ void DbSlice::PreUpdate(DbIndex db_ind, PrimeIterator it) {
   }
 
   it.SetVersion(NextVersion());
+  it->first.SetSticky(false);
```

Comment:
> anyway, in case of STICK, I do think it makes sense to destruct this property upon writes (besides DEL of course), so key seems like a sane choice to me.

You might have missed a not here. Then the besides and so make more sense 🤔